.eleventy.js

```diff
@@ -19,6 +19,14 @@ module.exports = function(eleventyConfig) {
     eleventyConfig.addPassthroughCopy({ "public/*": "." });
     eleventyConfig.addPassthroughCopy("docs/**/*.svg");
 
+    // Replace the default Markdown engine with a new one that creates ID in headings
+    // See: https://github.com/eslint/website/issues/747
+    const markdownIt = require("markdown-it");
+    const markdownItAnchor = require("markdown-it-anchor");
+    const markdownLib = markdownIt({ html: true }).use(markdownItAnchor);
+
+    eleventyConfig.setLibrary("md", markdownLib);
```

Comment:
This will overwrite `eleventyConfig.setLibrary("md")` from [_11ty/plugins/syntax-highlighting.js](https://github.com/eslint/website/blob/master/_11ty/plugins/syntax-highlighting.js)

For example: 

https://deploy-preview-755--eslint.netlify.app/docs/rules/for-direction (no syntax highlighting)

vs 

https://eslint.org/docs/rules/for-direction

Reply:
Nice catch. I’ll take a look. Can’t help but feel that Eleventy really overcomplicates this.